backend/controllers/github.go

```
@@ -53,64 +53,88 @@ type DiggerController struct {
 func (d DiggerController) GithubAppWebHook(c *gin.Context) {
 	c.Header("Content-Type", "application/json")
 	gh := d.GithubClientProvider
-	log.Printf("GithubAppWebHook")
+	slog.Info("Processing GitHub app webhook")
 
 	appID := c.GetHeader("X-GitHub-Hook-Installation-Target-ID")
 
 	_, _, webhookSecret, _, err := d.GithubClientProvider.FetchCredentials(appID)
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)